### PR TITLE
Expose the connectSid

### PR DESCRIPTION
**Pull request by Myztiq**
- [x] @Nathan219

### CODE REVIEW

File: lib/models/user.js [hunk 1]
```diff
@@ -94,6 +94,7 @@ User.prototype.githubLogin = function (accessToken, cb) {
           }
           self.id('me');
           self.fetch(cb);
+          self.connectSid = res.headers['set-cookie'][0].split(';')[0].split('=')[1];
```
Comment (Nathan219) [inline lib/models/user.js]: should this use keypather?

Comment (Myztiq) [inline lib/models/user.js]: Are we ever never going to send `set-cookie`?

Comment (Myztiq) [inline lib/models/user.js]: Also, if this fails it's the last thing in the list so if the stack dies it's totally fine

**Comment by Nathan219** [comment]
Awesome!

**Comment by Nathan219** [comment]
LGTM

**Pull request merged**